cdap-ui/app/cdap/components/Home/index.js

```diff
@@ -31,6 +31,8 @@ const defaultFilter = ['app', 'dataset', 'stream'];
 class Home extends Component {
```

Comment:
May not be now but we need to define what functions are available as part of the class. `processQueryString`, `handleSearch`, `makeFilterParams` not sure if these functions, accessible through `Home` instance would mean anything.